Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -847,6 +847,20 @@ MediaStream& GStreamerMediaEndpoint::mediaStreamFromRTCStream(String mediaStream
     return *mediaStream.iterator->value;
 }
 
+String GStreamerMediaEndpoint::trackIdFromSDPMedia(const GstSDPMedia& media)
+{
+    const char* msidAttribute = gst_sdp_media_get_attribute_val(&media, "msid");
+    if (!msidAttribute)
+        return emptyString();
+
+    GST_LOG_OBJECT(m_pipeline.get(), "SDP media msid attribute value: %s", msidAttribute);
+    auto components = String::fromUTF8(msidAttribute).split(' ');
```

Comment:
Maybe you can use a StringView here.

Reply:
It has no `fromUTF8()`, only `fromLatin1()`. It's unclear if StringView can handle UTF8.